src/sentry/static/sentry/app/views/eventsV2/landing.tsx

```diff
@@ -0,0 +1,197 @@
+  location: Location;
+  router: ReactRouter.InjectedRouter;
+  params: Params;
+};
+
+class DiscoverLanding extends React.Component<Props> {
+  static propTypes: any = {
+    organization: SentryTypes.Organization.isRequired,
+    location: PropTypes.object.isRequired,
+    router: PropTypes.object.isRequired,
+  };
+
+  state = {
+    isBannerHidden: localStorage.getItem('discover-banner-dismissed'),
+  };
+
+  getDocumentTitle = (name: string | undefined): Array<string> => {
+    return typeof name === 'string' && String(name).trim().length > 0
+      ? [String(name).trim(), t('Discover')]
+      : [t('Discover')];
+  };
+
+  handleClick = () => {
+    localStorage.setItem('discover-banner-dismissed', true);
+    this.setState({isBannerHidden: true});
+  };
+
+  renderBanner() {
+    const bannerDismissed = localStorage.getItem('discover-banner-dismissed');
+
+    if (bannerDismissed) {
+      return null;
+    }
+
+    const eventView = EventView.fromEventViewv1(DEFAULT_EVENT_VIEW_V1);
+
+    const to = {
+      pathname: location.pathname,
+      query: {
+        ...eventView.generateQueryStringObject(),
+      },
+    };
+
+    return (
+      <Banner
+        title={t('Discover')}
+        subtitle={t('Customize your query searches')}
+        onCloseClick={this.handleClick}
+      >
+        <Button
+          to={to}
+          onClick={() => {
+            trackAnalyticsEvent({
+              eventKey: 'discover_v2.prebuilt_query_click',
+              eventName: 'Discoverv2: Click a pre-built query',
+              organization_id: this.props.organization.id,
+              query_name: eventView.name,
+            });
+          }}
+        >
+          {t('Build a new query')}
+        </Button>
+      </Banner>
+    );
+  }
+
+  renderActions() {
+    return (
+      <StyledActions>
+        <StyledSearchBar />
+        <Button priority="primary">{t('Build a new query')}</Button>
+      </StyledActions>
+    );
+  }
+
+  renderNewQuery() {
+    const {location, organization} = this.props;
+
+    return (
+      <div>
+        {this.renderBanner()}
+        {DISPLAY_SEARCH_BAR_FLAG && this.renderActions()}
+        <QueryList location={location} organization={organization} />
+      </div>
+    );
+  }
+
+  renderQueryRename = (hasQuery: boolean, eventView: EventView) => {
+    if (!hasQuery) {
+      return null;
+    }
+
+    const {organization} = this.props;
+
+    return (
+      <div>
+        <EventInputName organization={organization} eventView={eventView} />
+      </div>
+    );
+  };
+
+  render() {
+    const {organization, location, router} = this.props;
+    const eventView = EventView.fromLocation(location);
+    const documentTitle = this.getDocumentTitle(eventView.name).join(' - ');
+
+    const hasQuery = eventView.isValid();
+
+    return (
+      <Feature features={['events-v2']} organization={organization} renderDisabled>
+        <DocumentTitle title={`${documentTitle} - ${organization.slug} - Sentry`}>
```

Comment:
Can we use `<SentryDocumentTitle>` here?

Reply:
Yep. 👍